incubator/react-native-error-trace-decorator/package.json

```
@@ -0,0 +1,53 @@
+{
+  "private": true,
+  "name": "@rnx-kit/react-native-error-trace-decorator",
+  "version": "0.0.1",
+  "description": "EXPERIMENTAL - USE WITH CAUTION - New package called react-native-error-trace-decorator",
+  "homepage": "https://github.com/microsoft/rnx-kit/tree/main/incubator/react-native-error-trace-decorator#readme",
+  "license": "MIT",
+  "author": {
+    "name": "Microsoft Open Source",
+    "email": "microsoftopensource@users.noreply.github.com"
+  },
+  "files": [
+    "lib/*"
+  ],
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "bin": "lib/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/rnx-kit",
+    "directory": "incubator/react-native-error-trace-decorator"
+  },
+  "engines": {
+    "node": ">=14.15"
+  },
+  "scripts": {
+    "build": "rnx-kit-scripts build",
+    "format": "rnx-kit-scripts format",
+    "lint": "rnx-kit-scripts lint",
+    "test": "rnx-kit-scripts test"
+  },
+  "devDependencies": {
+    "@rnx-kit/scripts": "*",
+    "@types/fs-extra": "^11.0.0",
+    "@types/minimist": "^1.2.0",
+    "eslint": "^8.0.0",
+    "jest": "^27.0.0",
+    "prettier": "^2.8.0",
+    "typescript": "^4.0.0"
+  },
+  "dependencies": {
+    "@rnx-kit/console": "*",
```

This should be set to a version:
```suggestion
    "@rnx-kit/console": "^1.0.11",
```

---

Updated